src/System.Windows.Forms.Design/tests/UnitTests/System/Windows/Forms/Design/ListControlStringCollectionEditorTests.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using Moq;
+using System.ComponentModel;
+
+namespace System.Windows.Forms.Design.Tests;
+
+public class ListControlStringCollectionEditorTests
+{
+    [Fact]
+    public void EditValue_WithNullContext_ReturnsBaseEditValue()
+    {
+        var editor = new ListControlStringCollectionEditor(typeof(string));
```

Comment:
```suggestion
        ListControlStringCollectionEditor editor = new(typeof(string));
```

Same in below tests as well